.eslintrc.js

```diff
@@ -30,6 +30,15 @@ module.exports = {
     'jsx-a11y/anchor-is-valid': 'off',
     'jsx-a11y/media-has-caption': 'off',
     'import/prefer-default-export': 'off',
-    'semi': 'warn'
+    'semi': 'warn',
+    "react/no-unknown-property": [
+      2,
+      {
+        "ignore": [
```

Comment:
thought(non-blocking): If this is added because our react uploader definitions are under-defined, we should probably follow up and fix it on the elements side.